pkg/vetter/conflictingvirtualservicehost/vet_test.go

```diff
@@ -420,16 +421,28 @@ var _ = Describe("Conflicting Virtual Service Host Vet Notes", func() {
 				},
 			}
 
+			expectedNote8 := &apiv1.Note{
+				Type:    vsHostNoteType,
+				Summary: vsHostSummary,
+				Msg:     vsHostMsg,
+				Level:   apiv1.NoteLevel_ERROR,
+				Attr: map[string]string{
+					"host":     "foo.com",
+					"routes":   "/foo exact /foo prefix",
+					"vs_names": "Vs4.bar, Vs4.bar",
```

Comment:
I think if there are conflicts within the same virtual service, then it shouldn't be listed twice in the note.

Reply:
Do you mean not list for each host? This was original code design is. I didn't know why each needs to be listed.

Reply:
I mean for the `vs_name`. When the conflicts are across virtual services, this makes it easier to track down what routes are causing conflicts. However, when they're in the same virtual service, there's only one virtual service, so it should be listed only once. 

I don't feel too strongly about this one, only do it if it's easy.